pyvista/core/utilities/fileio.py

```diff
@@ -993,6 +997,145 @@ def from_meshio(mesh: meshio.Mesh) -> UnstructuredGrid:
     return grid
 
 
+def to_meshio(mesh: DataSet) -> meshio.Mesh:
+    """Convert a PyVista mesh to a ``meshio`` mesh instance.
+
+    Parameters
+    ----------
+    mesh : pyvista.DataSet
+        Any PyVista mesh/spatial data type.
+
+    Returns
+    -------
+    meshio.Mesh
+        A mesh instance from the ``meshio`` library.
+
+    Raises
+    ------
+    ImportError
+        If the meshio package is not installed.
+
```

Comment:
Add cross-refs
```suggestion

    See Also
    --------
    from_meshio, read_meshio, save_meshio, :func:`~pyvista.wrap`

```

Would also be good to add a similar cross-refs to these other meshio methods in `fileio.py` but it's OK if not